test/scripts/create.test.js

```diff
@@ -66,6 +66,16 @@ contract('create script', function([_, owner]) {
     await assertProxy(data.proxies[contractAlias][0], { version: defaultVersion, say: 'V1', implementation });
   });
 
+  it('should record the proxy deployed address in contract build json file', async function () {
+    await createProxy({ contractAlias, packageFileName, network, txParams });
+
+    const contractData = fs.parseJson(`${process.cwd()}/build/contracts/${contractName}.json`)
+    const proxyData = fs.parseJson(networkFileName)
+    const proxyAddress = proxyData.proxies[contractAlias][0].implementation
+    const network_id = Contracts.getFromLocal(contractName).network_id
+    contractData.networks[network_id].address.should.be.eq(proxyAddress)
+  });
+
```

Comment:
I'd suggest adding a test to check that `Contract.deployed()` resolves correctly, since it's the actual feature we want to enable